src/realm/sync/transform.cpp

```diff
@@ -1570,11 +1600,17 @@ DEFINE_MERGE_NOOP(Instruction::SetClear, Instruction::EraseObject);
 
 DEFINE_NESTED_MERGE(Instruction::Update)
 {
+    using Type = Instruction::Payload::Type;
+
+    if (outer.value.type == Type::ObjectValue || outer.value.type == Type::Dictionary) {
```

Comment:
Is this a related issue or a separate issue?

Reply:
It's related.